render/src/shape_utils.rs

```diff
@@ -3,6 +3,9 @@ use enum_map::Enum;
 use smallvec::SmallVec;
 use swf::{CharacterId, FillStyle, LineStyle, Rectangle, Shape, ShapeRecord, Twips};
 
+/// Controls the accuracy of the approximated quadratic curve, when splitting up a cubic curve
```

Comment:
again, a minor nit: `... into quadratic ones` ?

Reply:
Feels a little tautological to me 😅

Reply:
Also maybe a unit for this number, or "meaning of the magnitude" would be nice to mention, but not critical.

Reply:
> Feels a little tautological to me 😅
Holy moly, I forgot the middle of the sentence by the time I got to its end... Why am _I_ code reviewing _after midnight_?